dependencies/src/main/resources/linkage-checker-exclusion-default.xml

```diff
@@ -132,4 +132,25 @@
       https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1296
     </Reason>
   </LinkageError>
+  <LinkageError>
+    <Source>
+      <Package name="io.netty.util.internal.logging" />
```

Comment:
It's package-level (not class-level), as the number of referencing class are not small.

https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1872